spec/cb/responses/industry/search_spec.rb

```diff
@@ -1,10 +1,20 @@
+# Copyright 2015 CareerBuilder, LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
 module Cb
   describe Responses::Industry::Search do
     let(:response) do
       { 'ResponseIndustryCodes' => {
-          'IndustryCodes' => {
-              'IndustryCode' => [{'like' => 'whoa'}]
-          }
+        'IndustryCodes' => {
+          'IndustryCode' => [{ 'like' => 'whoa' }]
+        }
       }
       }
```

Comment:
I don't know how much you want to fixup styling in this PR, but this one also seems weird with the double curly closing braces.